wallet/src/Pos/Wallet/Web/Methods/Payment.hs

```diff
@@ -137,6 +142,23 @@ instance
     getNewAddress (accId, passphrase) = do
         clientAddress <- L.newAddress RandomSeed passphrase accId
         decodeCTypeOrFail (cadId clientAddress)
+    -- We are creating a sample HD address here. It has the same
+    -- length (76) as the biggest one created by Daedalus.
+    getFakeChangeAddress = do
+        let passphrase = DBA.replicate 10 255
+        SecretKey sk <- snd <$> keyGen
+        encSK <- mkEncSecret passphrase sk
```

Comment:
Why didn't `emptyPassphrase` do it for you?

Reply:
Hm, I was trying hard to create an address with length 76 and was trying different passphrases in particular. I guess `emptyPassphrase` would work too, I will check.